agent/bench-scripts/templates/fio.job

```diff
@@ -24,7 +24,7 @@ write_hist_log = fio
 log_hist_msec = 10000
 # log_hist_coarseness = 4 # 76 bins
 
-[job-$target]
+[TheJob]
```

Comment:
This change concerns me since I run multi-job against a single host and am therefor given multiple jobs with unique names in the current scheme.  For example:

[global]
bs=4k
runtime=120
ioengine=libaio
iodepth=12
direct=1
sync=0
time_based=1
clocksource=gettimeofday
ramp_time=5
write_bw_log=fio
write_iops_log=fio
write_lat_log=fio
log_avg_msec=1000
write_hist_log=fio
log_hist_msec=10000
norandommap=1

[job-/dev/disk/by-id/nvme-INTEL_SSDPED1K375GA_PHKS8020006Z375AGN]
filename=/dev/disk/by-id/nvme-INTEL_SSDPED1K375GA_PHKS8020006Z375AGN
rw=read
size=10g
numjobs=1

[job-/dev/disk/by-id/nvme-INTEL_SSDPED1K375GA_PHKS802000W8375AGN]
filename=/dev/disk/by-id/nvme-INTEL_SSDPED1K375GA_PHKS802000W8375AGN
rw=read
size=10g
numjobs=1

This yields uniquely identifying information in the JSON results that would be missing without unique job names.  Why does that need to be removed?

Reply:
Karl, so it sounds like we could take out the change to fio.job.  This is exactly why I created a separate fio-shared-fs.job, because I didn't want to disrupt existing usage of fio.job, only I didn't realized significance of "[fio-$target]" to you.  It does have some weird behavior associated with it, but as long as you don't put slashes in the target I guess it would be ok in fio.job.  But for fio-shared-fs.job, the target isn't a particular block device or set of block devices, it is a directory, and there can be only 1, so we really don't want to have "[fio-$target]" there.

Reply:
I did this in latest commit to this PR.

Reply:
This has been resolved by `fio.job` no longer being changed.